UI/Web/src/app/want-to-read/want-to-read/want-to-read.component.html

```diff
@@ -28,7 +28,7 @@ <h6 subtitle>{{seriesPagination.totalItems}} Series</h6>
         </ng-template>
 
         <ng-template #noData>
-            No Series match your filter or exist in your list.
+            There are no items. Try adding a series.
```

Comment:
This can also show nothing if a filter is active. We could use filterActive to show different messages